api/tests/api_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from server import app as api
+
+
+@pytest.fixture()
+def app():
+    yield api
+
+
+@pytest.fixture()
+def client(app):
+    return app.test_client()
+
+
+def test_no_resource_given(client):
```

Comment:
An improved name for this test could be:
```suggestion
def test_status_code_when_no_resource_given(client):
```
or alternatively

```suggestion
def test_when_no_resource_given_then_status_code_404(client):
```
or something similar.